src/operators/validate_byte_range.cc

```diff
@@ -37,6 +37,11 @@ bool ValidateByteRange::getRange(const std::string &rangeRepresentation,
                 "' into a number");
             return false;
         }
+        if ((start < 0) || (start > 255)) {
+            error->assign("Invalid range start value: " +
```

Comment:
In this case we aren't checking a range but a single byte.
```suggestion
            error->assign("Invalid byte value: " +
```